cmd/skupper/skupper_network.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"fmt"
+	"github.com/skupperproject/skupper/pkg/utils/formatter"
+	"github.com/spf13/cobra"
+	"strings"
+)
+
+func NewCmdNetwork() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "network [command]",
+		Short: "Show information about the sites and services included in the network.",
+	}
+	return cmd
+}
+
+var selectedSite string
+
+func NewCmdNetworkStatus(newClient cobraFunc) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:    "status",
+		Short:  "Shows information about the current site, and connected sites.",
+		Args:   cobra.MaximumNArgs(1),
+		PreRun: newClient,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			silenceCobra(cmd)
+
+			sites, err := cli.NetworkStatus()
+
+			if err != nil {
+				fmt.Println("Unable to retrieve network information.")
+				return nil
+			}
+
+			siteConfig, err := cli.SiteConfigInspect(nil, nil)
+			if err != nil || siteConfig == nil {
+				fmt.Printf("The site configuration is not available: %s", err)
+				fmt.Println()
+				return nil
+			}
+
+			currentSite := siteConfig.Reference.UID
+
+			if sites != nil && len(sites) > 0 {
+				siteList := formatter.NewList()
+				siteList.Item("Sites:")
+				for _, site := range sites {
+
+					if site.Name != selectedSite && selectedSite != "all" {
+						continue
+					}
+
+					location := "remote"
+					siteVersion := site.Version
+
+					if len(site.MinimumVersion) > 0 {
+						siteVersion = fmt.Sprintf("%s (minimum version required %s)", site.Version, site.MinimumVersion)
+					}
+
+					if site.SiteId == currentSite {
+						location = "local"
+					}
+
+					newItem := fmt.Sprintf("[%s] %s - %s ", location, site.SiteId[:7], site.Name)
+
+					newItem = newItem + fmt.Sprintln()
+					detailsMap := map[string]string{"name": site.Name, "namespace": site.Namespace, "URL": site.Url, "version": siteVersion}
+
+					if len(site.Links) > 0 {
+						detailsMap["sites linked to"] = fmt.Sprint(strings.Join(site.Links, ", "))
```

Comment:
It would be easier to read if you could include the name as well. Example:
```
sites linked to: c3e2204 (site1)
```

Reply:
Nice feedback, thanks!